### PR TITLE
Misc

### PR DESCRIPTION
**Pull request by KaiVolland**
This adds some fixes and changes to i18n.
It also readds the `getIdMap` function.

### CODE REVIEW

File: config/pKeyAssociations.json [hunk 1]
```diff
@@ -0,0 +1,6 @@
+{
+  "User": "username",
+  "Application": "name",
+  "UserGroup": "name",
+  "Layer": "name"
```
Comment (marcjansen) [inline config/pKeyAssociations.json]: Hmm. Is there a reason not to use a numeric key? I can imagin people having multiple layers with the same name at least...

File: public/locale/de.json [hunk 2]
```diff
@@ -20,13 +20,17 @@
...
-    "logout": "Abmelden"
+  "Entities": {
+    "application": "Applikation",
+    "applications": "Applikationen",
+    "layer": "Thema",
+    "layers": "Themen",
+    "user": "Nutzer",
+    "users": "Nutzer",
+    "usergroup": "Nutzer Gruppe",
+    "usergroups": "Nutzer Gruppen",
```
Comment (marcjansen) [inline public/locale/de.json]: No spaces needed here and above

File: public/locale/de.json [hunk 3]
```diff
@@ -65,5 +75,45 @@
...
+    "fieldSeperator": "Feldtrenner",
+    "textSeperator": "Texttrenner",
+    "tabulator": "Tabulator",
+    "comma": "Komma",
+    "semicolon": "Semikolon",
+    "space": "Leerzeichen",
+    "singleQuote": "Einfache Anführungszeichen",
+    "doubleQuote": "Doppelte Anführungszeichen",
+    "csvFile": "CSV Datei",
+    "loadingSuccess": "{{count}} Entitäte(n) von {{modelName}} geladen.",
```
Comment (marcjansen) [inline public/locale/de.json]: Even better would it be if `Entitäte(n)` would be a placeholder and the logic would supply it with singular or plurar deoending on the count.

File: public/locale/de.json [hunk 4]
```diff
@@ -65,5 +75,45 @@
...
+  },
+  "ManagementGrid": {
+    "validation": {
+      "isRequired": "Feld {{field}} darf nicht leer sein.",
+      "isNotANumber": "Feld {{field}} muss eine Zahl enthalten.",
+      "isNotAnInteger": "Feld {{field}} muss eine Ganzzahl enthalten.",
+      "decimalSeperator": "Feld {{field}} muss den Dezimaltrenner '.' verwenden."
+    },
+    "csv": {
+      "unknownExperimentalFactors": "Invalides CSV. Die CSV-Datei enthält die unbekannten Versuchsfaktoren {{invalidFactors}}. Bitte legen Sie diese zuerst an.",
```
Comment (marcjansen) [inline public/locale/de.json]: Experimental features?

File: public/locale/de.json [hunk 5]
```diff
@@ -65,5 +75,45 @@
...
+    "importData": "Daten importieren",
+    "importing": "Importiert…",
+    "importSuccess": "{{count}} Entitäte(n) von {{modelName}} importiert.",
+    "deleteConfirm": "Wollen sie die selektierten Einträge wirklich löschen?",
+    "deleting": "Löscht…",
+    "deleteSuccess": "{{count}} Entitäte(n) von {{modelName}} gelöscht.",
+    "saveConfirm": "Wollen sie die Änderungen wirklich speichern?",
+    "saving": "Aktualisiert…",
+    "saveSuccess": "{{count}} Entitäte(n) von {{modelName}} aktualisiert.",
+    "featureEditorTitle": "Geometrie Editor"
```
Comment (marcjansen) [inline public/locale/de.json]: no space?

File: config/pKeyAssociations.json (hunk 1 above)
Comment (KaiVolland) [reply config/pKeyAssociations.json]: This association table specifies which property will be displayed in `Select` etc.

Comment (marcjansen) [reply config/pKeyAssociations.json]: ok, stupid me

**Pull request merged**